nodebuilder/p2p/flags.go

```diff
@@ -2,14 +2,20 @@ package p2p
 
 import (
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/multiformats/go-multiaddr"
 	"github.com/spf13/cobra"
 	flag "github.com/spf13/pflag"
 )
 
-var (
-	p2pMutualFlag = "p2p.mutual"
+const (
+	// DefaultNetwork is the default network of the current build.
+	DefaultNetwork   = Arabica
+	EnvCustomNetwork = "CELESTIA_CUSTOM"
+	nodeNetworkFlag  = "node.network"
```

Comment:
would call it `networkFlag`

Reply:
should p2pMutualFlag be called mutualFlag too? Or rather p2pNetworkFlag and p2pMutual flag together?

Reply:
i think the p2p is redundant - lets just make all flag names 

`networkFlag` `mutualFlag`

Reply:
since the flags are defined per-subpkg

Reply:
done